config/crd/patches/webhook_in_metallbs.yaml

```diff
@@ -3,7 +3,7 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: metallbs.metallb.quay.io/fpaoline/metallboperator
+  name: metallbs.metallb.quay.io/metallb/metallb-operator
```

Comment:
the above also supposed to be the API not the image we need to undo those and use metallbs.metallb.io as it was originally auto generated when the new API was added